castervoice/lib/ccr/core/nav.py

```diff
@@ -216,8 +216,11 @@ class Navigation(MergeRule):
         'save':
             R(Key("c-s"), rspec="save", rdescript="Core: Save"),
         'shock [<nnavi50>]':
-            R(Key("enter"), rspec="shock", rdescript="Core: Enter")* Repeat(extra="nnavi50"),
-
+            R(Key("enter"), rspec="shock", rdescript="Enter")* Repeat(extra="nnavi50"),
+        'hard shock [<nnavi50>]':
+            R(Key("s-enter"), rspec="hard shock", rdescript="Hard Shock")* Repeat(extra="nnavi50"),
+        'hard deli [<nnavi50>]':
+            R(Key("s-del"), rspec="hard deli", rdescript="Hard Deli")* Repeat(extra="nnavi50"),
```

Comment:
Shouldn't these be "shin shock" and "shin deli"?

Reply:
`s-del` does nothing in many apps on my system and acts like backspace in RStudio. @alexboche, where do you use it?